read-the-docs/source/Usage.rst

```diff
@@ -0,0 +1,74 @@
+Usage
+=====
+
+Create a Ticket object
+----------------------
+
+Create a JiraTicket, RTTicket, RedmineTicket, BugzillaTicket
+or ServiceNowTicket object with ``<url>``, ``<project>`` and ``<auth>``. This
+verifies that you are able to properly authenticate to the ticketing tool.
+To use HTTP Basic Authentication, the ``<auth>`` parameter should contain the
+username and password specified as a tuple. For tools that support kerberos
+authentication (JIRA and RT), the ``<auth>`` parameter should contain
+'kerberos'.
+
+
+Work with a new ticket
+----------------------
+
++ Create a ticket with the ``create()`` method. This sets the ``ticket_id``
+  instance variable, allowing you to perform more tasks on the ticket.
+
++ Add comments, edit ticket fields, add watchers, change the ticket
+  status, etc on the ticket.
+
++ Close ticket Requests session with ``close_requests_session()``.
+
++ To work on existing tickets, you can also pass in a fourth parameter
```

Comment:
Lines 27 - 38 describe working with existing tickets. This should be moved under the "Work with an existing ticket" section.